microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/services/MicroProfileValidator.java

```diff
@@ -227,6 +232,7 @@ private static boolean isIntegerString(String str) {
 	}
 
 	private static boolean isFloatString(String str) {
+		if (str == null || str.isEmpty()) return false;

```

Comment:
please always use { } for if statements. See https://blog.codecentric.de/en/2014/02/curly-braces/

Reply:
We should provide in `com.redhat.microprofile.utils.StringUtils.isEmpty(String s)` like https://github.com/apache/commons-lang/blob/e389ce1edcc909444944dcb4064050f4eab7b872/src/main/java/org/apache/commons/lang3/StringUtils.java#L3582

because there are a lot of code which does that in MicroProfileValidator (and perhaps in an another classes?).

Reply:
@fbricon cool article, fixed 
@angelozerr maybe we use this method? https://github.com/redhat-developer/quarkus-ls/blob/bdf6c5fc10f3fe03ad3d708086551739abb1348d/microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/utils/StringUtils.java#L49

Reply:
This method return true if there are some spaces in the string. I'm not sure that it's the expected behavior.

Reply:
No it returns true if there are non-whitespace characters

Reply:
So yes, StringUtils.hasText(String) does the job

Reply:
If you want to use hasText, let's go, but it will change the original behavior because this method returns true if there are only spaces (it's a kind of trim).

Reply:
no it doesn't

Reply:
Updated to use `StringUtils.hasText(String) `